feature-libs/user/account/components/login-form-oidc/login-form-oidc.module.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, NotAuthGuard, provideDefaultConfig } from '@spartacus/core';
+import { SpinnerModule } from '@spartacus/storefront';
+import { LoginFormOidcComponent } from './login-form-oidc.component';
+
+@NgModule({
+  imports: [CommonModule, RouterModule, SpinnerModule],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ReturningOIDCLoginComponent: {
```

Comment:
this component will be used for both login and return, so the name `ReturningOIDCLoginComponent` is not accurate